.goreleaser.yml

```
@@ -0,0 +1,58 @@
+archives:
+  - builds:
+      - jb
+    format: tar.gz
+    format_overrides:
+      - format: zip
+        goos: windows
+    name_template: '{{ .ProjectName }}-{{ .Os }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}-{{ .Mips }}{{ end }}'
+before:
+  hooks:
+    - go mod tidy
+    - go generate ./...
+brews:
+  - description: The jsonnet-bundler is a package manager for Jsonnet.
```

```suggestion
  - description: jsonnet-bundler is a package manager for Jsonnet.
```

I think we usually refer to it without a "the"

---

This section will go away entirely since there's no self-hosting of the Homebrew formula.

---

Pushed just now.